src/controllers/midi/midicontroller.cpp

```diff
@@ -159,42 +159,49 @@ void MidiController::destroyOutputHandlers() {
     }
 }
 
-QString formatMidiMessage(unsigned char status, unsigned char control, unsigned char value,
-                          unsigned char channel, unsigned char opCode) {
+QString formatMidiMessage(unsigned char status, unsigned char control,
+                          unsigned char value, unsigned char channel,
+                          unsigned char opCode, mixxx::Duration timestamp) {
     switch (opCode) {
         case MIDI_PITCH_BEND:
-            return QString("MIDI status 0x%1: pitch bend ch %2, value 0x%3")
+            return QString("MIDI status 0x%1: pitch bend ch %2, value 0x%3, timestamp %4")
```

Comment:
I did not actually tested it, but in my other polling branch these lines are exceeding 80 colums, leading to a line break in the consoles. This makes the output harder to read. Maybe we can remove some less significant information to avoid line breaks. 
I would also keep the same info at the same column, Which means here Timestamp first.